BrWizard.vue

```diff
@@ -63,20 +35,23 @@
  */
 'use strict';
 
+import BrWizardNavigation from './BrWizardNavigation.vue';
```

Comment:
bikeshed: `BrWizardNavigator` sounds better to me ... thoughts?

Reply:
No strong preference, I lifted the name from Quasar's QStepper components. I'll go ahead and rename `BrWizardNavigation` to `BrWizardNavigator`.